docs/lib/examples/FormFeedback.js

```diff
@@ -20,8 +20,12 @@ export default class Example extends React.Component {
         <FormGroup>
           <Label for="examplePassword">Invalid input</Label>
           <Input invalid />
-          <FormFeedback>Oh noes! that name is already taken</FormFeedback>
-          <FormText>Example help text that remains unchanged.</FormText>
+          <FormFeedback tooltip>Oh noes! that name is already taken</FormFeedback>
```

Comment:
Can you update the examples so that there is what was previously there (an input with no validation, an input with valid feedback, and an input with invalid feedback) to add an input with valid tooltip feedback, and an input with invalid tooltip feedback.

right now the one example is not working because the field is marked as invalid and the tooltip is set to be valid (this mis-match breaks the display css within bootstrap). Also I don't want to replace the current feedback examples, but instead extend the example to include these new variants